projects/packages/search/changelog/update-search-record-meter-notice-box-fix

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: changed
+
+updated notice box content when site is not indexed
```

Comment:
```suggestion
Record Meter: updated notice box content when site is not indexed
```

Based on the P2 reply, we should reference the internal component we're adjusting but don't have to mention Search :)